src/GeniusYield/Test/Privnet/Examples/Oracle.hs

```diff
@@ -20,83 +20,84 @@ import           GeniusYield.OnChain.Examples.ReadOracle.Compiled
 import           GeniusYield.Test.Privnet.Asserts
 import           GeniusYield.Test.Privnet.Ctx
 import           GeniusYield.Test.Privnet.Setup
-import           GeniusYield.TxBuilder.Class
+import           GeniusYield.TxBuilder
 import           GeniusYield.Types
 
 readOracleValidatorV2 :: GYValidator 'PlutusV2
 readOracleValidatorV2 = validatorFromPlutus readOracleValidator
 
 tests :: Setup -> TestTree
 tests setup = testGroup "oracle"
-    [ testCaseSteps "without-ref" $ \info -> withSetup setup info $ \ctx -> do
+    [ testCaseSteps "without-ref" $ \info -> withSetup info setup $ \ctx -> do
```

Comment:
`withSetup` order changed due to the addition of `withSetup'` - there is now better logging capabilities in the privnet testing code. You might consider adding a flag to the testing code to choose the logging level. If you do, use `withSetup'` with said level. `withSetup` defaults to `GYInfo`.

`withSetupOld` can be used for migration if changing the argument order seems too painful immediately.